cmd/client/wallet/lib/lib.go

```diff
@@ -2,7 +2,7 @@ package lib
 
 import (
 	"fmt"
-	"time"
+	//	"time"
```

Comment:
remove it if it is not needed.

Reply:
the wallet part is not fixed yet. I am currently switching between hard code peer info with p2p discovery